.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -0,0 +1,54 @@
+on:
```

Comment:
This is a shared repo and this config file is only for spanner. So the file name might better to be `spanner-integration-tests-against-emulator.yaml` with a prefix `spanner-`.